app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -17,7 +19,12 @@ def resource_locks
       end
 
       def plan(job_invocation, host, template_invocation, proxy_selector = ::RemoteExecutionProxySelector.new, options = {})
-        action_subject(host, :job_category => job_invocation.job_category, :description => job_invocation.description, :job_invocation_id => job_invocation.id)
+        features = template_invocation.templateremote_execution_features.pluck(:label).uniq
```

Comment:
Uh-oh @lzap, @adamruzicka this doesn't seem to be correct :hot_face:

Reply:
Yeah, just noticed it as well https://github.com/theforeman/foreman_remote_execution/pull/595